components/sup/tests/utils/mod.rs

```diff
@@ -120,3 +125,46 @@ where
         }
     }
 }
+
+fn write_default_svc_user_and_group_metafiles<P>(hab_pkg_path: P)
+where
+    P: AsRef<Path>,
+{
+    let svc_user_metafile = hab_pkg_path.as_ref().join(MetaFile::SvcUser.to_string());
```

Comment:
Given that other metadata file path information is encapsulated as functions of `HabRoot`, what do you think about adding a `svc_user_file` and `svc_group_file` to that module and calling them here?

(Aside: at some point, I think I'd like something like `HabRoot` to be part of the "real" code)

Reply:
I refactor as per your suggestion. It made the calling function require more context and I'm not sure I like this version better, but it puts the metafile-computing logic in with `HabRoot` which makes sense.